jujugui/static/gui/src/app/state/state.js

```diff
@@ -82,6 +95,76 @@ const State = class State {
   }
 
   /**
+    Check the supplied block of path to see if this is a user store path or
+    not. These options are becuase the store path can come in many forms and
+    we need to know if it's a user store path for both parsing and
+    generating the path.
+    @return {Boolean}
+  */
+  _isUserStorePath(block) {
```

Comment:
This method is used for both parsing of the path and generating the path from state so in the latter case there is no 'u'